### PR TITLE
CAMEL-10207: Fixed test configuration for Bam. Added repetition in ca…

### PR DESCRIPTION
**Pull request by nicolaferraro**
…se of failure.

```
Results :

Tests run: 224, Failures: 0, Errors: 0, Skipped: 0
```

### CODE REVIEW

**Comment by davsclaus** [comment]
Thanks for the PR it has been merged. Do you mind closing this?

File: tests/camel-itest-spring-boot/src/test/resources/components/bam-persistence.xml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ Copyright 2006 The Apache Software Foundation.
```
Comment (davsclaus) [inline tests/camel-itest-spring-boot/src/test/resources/components/bam-persistence.xml]: Can you use the same license header as the other XML files in Apache Camel. They need to be similar as we have a RAT check that check the code for license alignments.

File: tests/camel-itest-spring-boot/src/test/resources/components/bam-spring.xml [hunk 2]
```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+	<!--
```
Comment (davsclaus) [inline tests/camel-itest-spring-boot/src/test/resources/components/bam-spring.xml]: Same story, copy license header from existing working xml file such as from a pom.xml file

**Comment by nicolaferraro** [comment]
Yeah, I copied that files from the test sources of Bam, so probably the header was missing there too.
I'm going to add it.

**Comment by nicolaferraro** [comment]
I'm going to open another PR.

**Pull request closed without merging**